Cauldron/Assets/Scripts/InputCheatManager.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.InputSystem;
+
+[Serializable]
+public class CheatCodes
+{
+    public string CheatCode;
+    public UnityEvent<string> Event;
+}
+
+public class InputCheatManager : MonoBehaviour
+{
+    [SerializeField]
```

Comment:
If this is handy for debug, you can mark it [Readonly] with Naughty attributes (check them out, it's an awesome little package of inspector goodies, free to use, we have it in the project).